pkg/controller/migmigration/annotation.go

```diff
@@ -333,6 +337,38 @@ func (t *Task) labelServiceAccounts(client k8sclient.Client, serviceAccounts Ser
 	return nil
 }
 
+// Add label to ImageStreeams
+func (t *Task) labelImageStreams(client compat.Client) error {
+	for _, ns := range t.sourceNamespaces() {
+		imageStreamList := imagev1.ImageStreamList{}
+		options := k8sclient.InNamespace(ns)
+		err := client.List(context.TODO(), options, &imageStreamList)
+		if err != nil {
+			log.Trace(err)
+			return err
+		}
+		for _, is := range imageStreamList.Items {
+			if is.Labels == nil {
+				is.Labels = map[string]string{}
+			}
+			is.Labels[IncludedInStageBackupLabel] = t.UID()
+			err = client.Update(context.Background(), &is)
+			if err != nil {
+				// TODO: confirm with the team if we want to collect errors
```

Comment:
I think we'd resolved the collecting errors question for now, so we can probably remove the TODO comment.

Reply:
added an issue and removed